### PR TITLE
MINOR: fix code listings connect.html

### PR DESCRIPTION
**Pull request by jlprat**
Uses the right tool to show code listings, so it is consistent with the
rest of the examples

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by jlprat** [comment]
CC. @cadonna as you reviewed the last doc changes on formatting and styling, you might want to review these as well. I split the changes in several PRs so it's more bearable to review.

**Review by showuon — APPROVED** [review]
@jlprat , LGTM. Thanks for the patch.

**Comment by jlprat** [comment]
Thanks for the review @showuon !

**Comment by mimaison** [comment]
Is there a reason why we never merged this?

**Comment by jlprat** [comment]
Hi @mimaison, @showuon  wasn't a committer back then I believe. I can rebase and get it committed. I forgot about this one actually

**Comment by mimaison** [comment]
I personally prefer `$` than `>` for the shell prompt. It's also what we use in the quickstart, so not sure if we should change it.

**Comment by jlprat** [comment]
I can use `$` instead, no problem. I'm rebasing, but it will take a minute, some changes happened in the meantime.

**Comment by jlprat** [comment]
@mimaison I updated the PR

File: docs/connect.html [hunk 1]
```diff
@@ -1042,21 +1003,20 @@ <h3><a id="connect_administration" href="#connect_administration">8.4 Administra
...
-    },
-    "tasks": [
-        {
-        "id": 0,
-        "state": "RUNNING",
-        "worker_id": "192.168.1.209:8083"
-        }
-    ]
-}</pre>
+    <pre class="line-numbers"><code class="language-json">{
```
Comment (mimaison) [inline docs/connect.html:1006]: The alignment seems broken for this code block:
image

**Review by mimaison — APPROVED** [review]
Thanks, LGTM apart from a small formatting issue with a block a JSON

File: docs/connect.html (hunk 1 above)
Comment (jlprat) [reply docs/connect.html:1006]: fixed now: 
image

(previous deleted message contained the wrong snapshot)

Comment (mimaison) [reply docs/connect.html:1006]: Thanks!

**Comment by jlprat** [comment]
No need to wait for CI as these changes are not relevant for CI

**Pull request merged**